src/babylon.scene.ts

```diff
@@ -711,6 +711,7 @@
         private _debugLayer: DebugLayer;
 
         private _depthRenderer: DepthRenderer;
+        private _geometryRenderer: GeometryRenderer;

```

Comment:
In current implementation the renderer renders all active meshes
It could be useful to allow devs to define other  renderes with different renderlist